tests/sles4sap/saptune/mr_test.pm

```diff
@@ -85,14 +85,14 @@ sub test_bsc1152598 {
     my $SLE = is_sle(">=15") ? "SLE15" : "SLE12";
 
     assert_script_run "mr_test verify Pattern/${SLE}/testpattern_bsc1152598#1_1";
-    assert_script_run 'echo -e "[version]\n# foobar-NOTE=foobar CATEGORY=foobar VERSION=0 DATE=foobar NAME=\" foobar \"\n[block]\nIO_SCHEDULER=none, foobar\n" > /etc/saptune/extra/scheduler-test.conf';
-    assert_script_run "saptune note apply scheduler-test";
+    assert_script_run '[version]\n# foobar-NOTE=foobar CATEGORY=foobar VERSION=0 DATE=foobar NAME=\" foobar \"\n[block]\nIO_SCHEDULER=noop, none, foobar\n" > /etc/saptune/extra/scheduler_test.conf';
```

Comment:
no `echo` here? I think that command called from an `assert_script_run()` will break the test. Is there a verification run for this particular case?

Reply:
> no `echo` here? I think that command called from an `assert_script_run()` will break the test. Is there a verification run for this particular case?

Good catch!

Reply:
Fixed. It wasn't caught because this particular routine runs only on baremetal.